apps/researcher/src/messages/en/messages.json

```diff
@@ -172,35 +172,35 @@
     "withholdingLabel": "The Withholding Notice is being used to indicate that no reproductions of this item or collection should ever be made. If there are already existing photographs or reproductions, these should not be used in any exhibition or publication in print, online, or any other format."
   },
   "LocalContextsNoticesOverview": {
-    "title": "Local Context Notices",
-    "description": "The Local Context notices support the inclusion of local protocols for access and use to cultural heritage that is digitally circulating outside community contexts. <link> Learn more about the Local Context notices.</link>.",
+    "title": "Local Contexts Notices",
+    "description": "The Local Contexts Collections Care (CC) Notices are visible identifiers developed specifically for institutions and repositories to support proper cultural care and stewardship of collections, led and guided by Indigenous protocols and authority. <link> Learn more about the Local Contexts Collections Care Notices.</link>.",
     "defaultProvider": "Colonial Collections Consortium",
-    "defaultLabel": "This is a default notice if no other labels or notices are assigned.",
+    "defaultLabel": "This is a default Notice in case no other Notices are assigned.",
```

Comment:
Zie opmerking NLs versie.

Reply:
ik maak hiervan: "This is a standard Notice added to each object to indicate an incomplete, incorrect and/or missing attribution." net zoals in de NLse versie